static/css/dark-mode.css

```
@@ -0,0 +1,1048 @@
+
+[data-theme="dark"] a:hover {
+  color: #7ccaff;
+}
+
+/* ===== 暗色模式：选中文本 ===== */
+[data-theme="dark"] ::selection {
+  background: rgba(74, 163, 223, 0.4);
+  color: #fff;
+}
+
+[data-theme="dark"] ::-moz-selection {
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed selector "::-moz-selection" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)